service/src/client.ts

```diff
@@ -453,7 +455,8 @@ export class Client extends EventEmitter {
     const config: Client.TransportConfigOptions = {
       host: decodedUri,
       keyName: sas.skn,
-      sharedAccessSignature: sas.toString()
+      sharedAccessSignature: sas.toString(),
```

Comment:
Why isn't there a fromToken in here and jobClient?

Reply:
Added, fixed